packages/compiler-vapor/src/transforms/transformText.ts

```diff
@@ -129,8 +143,14 @@ function processTextContainer(
   children: TextLike[],
   context: TransformContext<ElementNode>,
 ) {
-  const values = children.map(child => createTextLikeExpression(child, context))
+  const values = children
+    .map(child => createTextLikeExpression(child, context))
```

Comment:
This logic is duplicated with the code above (line 103). Perhaps we can extract a function that also incorporates the logic of `createTextLikeExpression`.

- logic reusability
- only traverse the children once

```ts
function processTextLikeChildren(children:TextLike[]):SimpleExpressionNode[]{
  const exps:SimpleExpressionNode[]=[]
  for(child ... children){
    markNonTemplate(node, context)
    // skip empty node
    if(!node.content) continue;

    if (node.type === NodeTypes.TEXT) {
      exps.push(...)
    } else {
      exps.push(...)
    }
  }

  return exps
}
```